src/evidently/metrics/data_drift/embeddings_drift.py

```diff
@@ -0,0 +1,113 @@
+from typing import Callable
+from typing import List
+from typing import Optional
+
+import numpy as np
+import umap
+
+from evidently.base_metric import InputData
+from evidently.base_metric import Metric
+from evidently.base_metric import MetricResult
+from evidently.metrics.data_drift.embedding_drift_methods import model
+from evidently.model.widget import BaseWidgetInfo
+from evidently.renderers.base_renderer import MetricRenderer
+from evidently.renderers.base_renderer import default_renderer
+from evidently.renderers.html_widgets import CounterData
+from evidently.renderers.html_widgets import WidgetSize
+from evidently.renderers.html_widgets import counter
+from evidently.renderers.html_widgets import plotly_figure
+from evidently.utils.visualizations import get_gaussian_kde
+from evidently.utils.visualizations import plot_contour
+
+SAMPLE_CONSTANT = 2500
+
+
+class EmbeddingsDriftMetricResults(MetricResult):
+    class Config:
+        dict_exclude_fields = {}
```

Comment:
probably should look like 
```
        dict_exclude_fields = {"plot_ref", "plot_cur",}
        field_tags = {k: {IncludeTags.Render} for k in dict_exclude_fields}
```

Reply:
(I assume it's render data since it's named plot)

Reply:
There is no IncludeTags in this branch. We could add this after merge?